gnd/build.gradle

```diff
@@ -130,13 +131,17 @@ dependencies {
     implementation "com.google.maps.android:android-maps-utils:0.5"
 
     // Firebase and related libraries.
+    implementation "com.google.firebase:firebase-analytics:$project.firebaseCoreVersion"
     implementation "com.google.firebase:firebase-core:$project.firebaseCoreVersion"
     implementation "com.google.firebase:firebase-firestore:$project.firebaseFirestoreVersion"
     implementation "com.google.firebase:firebase-auth:$project.firebaseAuthVersion"
     implementation "com.google.firebase:firebase-perf:$project.firebasePerfVersion"
     implementation "com.github.FrangSierra:RxFirebase:$project.rxFirebaseVersion"
     implementation 'com.google.firebase:firebase-storage:19.1.0'
 
+    // Crashlytics
```

Comment:
To note in the future: the name of the library is already clear from the build artifact name. In general we only want to add comments that clarify details not obvious from code, since redundant comments decrease the signal-to-noise ratio.